docs-source/content/samples/simple/domains/model-in-image/initial.md

```diff
@@ -262,7 +262,11 @@ If you don't see the `weblogic-deploy.zip` file, then you missed a step in the [
 Now, you use the Image Tool to create an image named `model-in-image:WLS-v1` that's layered on a base WebLogic image. You've already set up this tool during the prerequisite steps.
 
 Run the following commands to create the model image and verify that it worked:
+Note: If you are creating a WebLogic model-in-image, add --chown oracle:root to imagetool.sh command. When creating FMW Infrastructure images, do not use the --chown option.
 
+{{% notice note %}}
+If you are taking the JRF path through the sample, then remove --chown oracle:root from the imagetool.sh command below. 
+{% /notice %}}
```

Comment:
Include markup back-quotes around each of `JRF`, `--chown oracle:root`, and `imagetool.sh`, and change {% /notice %}} to {{% /notice %}}.   

If you want to see if the .md if formatting is as expected, go to docs-source, and run the 'runpublic.sh' command.   This will start a hugo web server with your version of the documentation which you can access at `http://$(hostname).$(dnsdomainname):1313`.

Reply:
Tom, I made the changes you requested. The note I added in update3.md looks different from the one in initial.md. I see that update3.md adds Note: differently does not use the notify note markup. Can you take a look. Let me know if we want it keep it uniform in update3.md.